desktop-exporter/app/components/empty-state-view/empty-state-view.tsx

```diff
@@ -0,0 +1,80 @@
+import React from "react";
+import {
+  Alert,
+  AlertIcon,
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader,
+  Flex,
+  Heading,
+  Image,
+  ListItem,
+  OrderedList,
+  Stack,
+  Text,
+} from "@chakra-ui/react";
+
+export function EmptyStateView() {
+  return (
+    <Flex
+      flexDirection="column"
+      align="center"
+      width="100%"
+      overflowY="scroll"
+    >
+      <Alert
+        status="info"
+        variant="solid"
+        minHeight="30px"
+      >
+        <AlertIcon />
+        No data yet. Checking again in 15 seconds...
```

Comment:
15 seconds is probably too slow given reloads are so quick. Maybe drop this to 5?